common-osd-overview.rst: add callsign panel and fonts from SD (#3173)
ArduPilot/ardupilot_wiki: common/source/docs/common-osd-overview.rst

- ``2`` = Betaflight.  bf- / inav-osd default style
- ``3`` = Bold. A bolder version of Betaflight
- ``4`` = Digital. 80's LED clock radio style

> .. note::    In firmware 4.1 and above onboard font options might be limited on 1 MB boards to save memory for other features. To make up for that, font upload from file system (SD card) is supported. This allows to choose from up to 10 font files (named "font0.bin" - "font9.bin") placed in the root directory of your SD card. Additionally this allows to easily add individualized and / or customized language charsets to your onboard OSD without the need to compile an individual firmware. Runtime font handling using parameter :ref:`OSD_FONT<OSD_FONT>` remains unaffected. MAVftp might be a convenient option to upload files to your SD card without having to physically remove it from your flight controller. 

Reviewer: ```suggestion
.. note::    In firmware 4.1 and above, onboard font options might be limited on 1 MB boards to save memory for other features. However, font upload from SD card is supported. This allows the user to choose from up to 10 font files (named "font0.bin" - "font9.bin") placed in the root directory of your SD card. Additionally, this allows the user to easily add individualized and/or customized language charsets for the onboard OSD without the need to compile  individual firmware. Runtime font changing using the parameter :ref:`OSD_FONT<OSD_FONT>` remains unaffected. MAVftp might be a convenient option to upload files to your SD card without having to physically remove it from your flight controller. 
```
